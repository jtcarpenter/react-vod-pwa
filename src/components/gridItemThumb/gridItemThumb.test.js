import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import {GridItemThumb} from
    'components/gridItemThumb/GridItemThumb.jsx';

const mockGridItemThumb = (
    <GridItemThumb />
)

describe('GridItemThumb', () => {
    it('should render', () => {
        const wrapper = shallow(mockGridItemThumb);
        expect(wrapper.exists()).toBe(true);
    });

    it('should display correctly', () => {
        const gridItemThumb = renderer.create(mockGridItemThumb);
        expect(gridItemThumb.toJSON()).toMatchSnapshot();
    });
});
