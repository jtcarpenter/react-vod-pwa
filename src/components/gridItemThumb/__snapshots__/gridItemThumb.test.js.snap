// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GridItemThumb should display correctly 1`] = `
<img
  className="sc-bdVaJa hqjwmh"
/>
`;
